llama_stack/providers/tests/inference/fixtures.py

```diff
@@ -9,7 +9,7 @@
 import pytest
```

Comment:
Follow up: rebase and update eval_task/datasets/scoring_functions fixtures.

Reply:
I fixed them now